src/Components/Search/Search.test.js

```diff
@@ -2,9 +2,26 @@ import React from 'react';
 import Search from './Search';
 import { render, screen } from '@testing-library/react';
 import '@testing-library/jest-dom';
+import { combineReducers, createStore } from 'redux';
+import { Provider } from 'react-redux';
+import allReducers from '../../reducers/index.js';
+
 
 describe('Search', () => {
-  it('Should pass an initial test', () => {
-    expect(true).toBe(true);
+  it('Should render a search form', () => {
+
+    const store = createStore(allReducers);
+
+    render(
+      <Provider store={store}>
+        <Search />
+      </Provider>
+    )
+
+    const dropDown = screen.getByRole('combobox')
```

Comment:
Awesome job figuring out how to test the dropdown & that it has the correct length!